### ISSUE TITLE
Importing global optionset translations throws exception after latest update

### ISSUE THREAD

**Issue by AntiHero7** [MscrmTools/MsCrmTools.Translator#49]
Hi, this exception is thrown when importing:
![Exception](https://user-images.githubusercontent.com/47024125/152759400-afa80ad1-4c4a-46c4-9501-08fdeb4a6099.png)

I tried it on other machine with some previous version of this plugin and the import of the same file was ok.

Could you please check it? Thanks in advance.

**Comment by MscrmTools** [comment]
Hi @AntiHero7, can you tell me if you have a global optionset that looks like a Boolean?

**Comment by AntiHero7** [comment]
Hi, maybe we do, but I would have to check. But the same optionsets were imported just fine on earlier version of the plugin, so I assumed that there were some code changes that caused this. 

So the reason might be that some global optionsets have only two options and that does not work well with the latest update? 
If we have some as global opSets, what could we do to make it work again? Thanks you.

**Comment by MscrmTools** [comment]
Yes, there was definitively a code change that impact global optionset but I wasn't expecting a global optionset to be of type Boolean (which seems to be the case according to the error message).
Unfortunately, I'm on vacation for 8 days so I won't be able to fix this until then.
I will unlist latest version so that it does not prompt to update

**Comment by AntiHero7** [comment]
Great, thanks a lot.

**Comment by MscrmTools** [comment]
@AntiHero7, can you test with this version and tell me if the issue is fixed ?
[MsCrmTools.Translator.zip](https://github.com/MscrmTools/MsCrmTools.Translator/files/8095716/MsCrmTools.Translator.zip)

**Comment by AntiHero7** [comment]
Hi, just tried it with global optionsets and it works. Thank you very much, should I close this Issue now?

**Comment by MscrmTools** [comment]
I will close it myself after having released the new version

**Comment by snake93** [comment]
I'm tring to import a file to translate entities but I'm receiving this error from different machines
![image](https://user-images.githubusercontent.com/12747888/154983733-0478ad64-4456-4bba-ab44-297c1992b12d.png)

**Comment by MscrmTools** [comment]
@snake93 this seems to be a different error
Can you test with this version?
[MsCrmTools.Translator.zip](https://github.com/MscrmTools/MsCrmTools.Translator/files/8110692/MsCrmTools.Translator.zip)

**Comment by snake93** [comment]
@MscrmTools Thank you for your sharing, but I have same error 
![image](https://user-images.githubusercontent.com/12747888/154998882-a3e3cd30-9a68-49a2-b545-0e0d152cda72.png)

**Comment by MscrmTools** [comment]
I'm sorry but I really do not reproduce. Any chance you can share your Excel file?

**Comment by snake93** [comment]
@MscrmTools please find attached the Excel file

[TranslateEntities.xlsx](https://github.com/MscrmTools/MsCrmTools.Translator/files/8115083/TranslateEntities.xlsx)

**Comment by snake93** [comment]
@MscrmTools unfortunately, I have the same problem

**Comment by MscrmTools** [comment]
@snake93 As per your error message, this is the only possibility. Did you remove juste the cell content or did you delete the row?

**Comment by snake93** [comment]
@MscrmTools it worked thank you

**Comment by MscrmTools** [comment]
ok, happy to know the problem was with the file. I will add some additional controls to avoid this scenario
